### DISCUSSION TITLE
Complexity Score calculation/ Estimation

### DISCUSSION THREAD

**Discussion by cvkonduru** [Cyfrin/foundry-full-course-cu#2313] [Q&A]
How to estimate the complexity score of a smart contract audit? I just started Auditing course, and didn't complete it yet and I am doing a deep dive of everything that Patrick is teaching. The technique might be in the other parts of the course that I didn't check out yet.But please let me know how to calculate or estimate it.

**Comment by EngrPips** [comment]
You can use tools such as solidity metrics to help with the calculation.

**Reply by cvkonduru** [reply]
I see, I tried it last night, it didn't give me a concrete number like for example 80%  anywhere in the report. Thanks in advance.

**Reply by EngrPips** [reply]
I am not sure what you mean by `80% anywhere in the report`
